tools/system_libs.py

```diff
@@ -1138,12 +1138,22 @@ class libasan_rt_wasm(SanitizerLibrary):
   src_dir = ['system', 'lib', 'compiler-rt', 'lib', 'asan']
 
 
-class libstandalonewasm(Library):
+class libstandalonewasm(MuslInternalLibrary):
   name = 'libstandalonewasm'
```

Comment:
Can you add some documentation here?   Whats the point of this library and why the time and string functions are being included?

Reply:
Adding.